Entity/ContentType.php

```diff
@@ -1768,4 +1775,15 @@ protected function deserializeProperty(string $name, $value)
                 parent::deserializeProperty($name, $value);
         }
     }
+
+    public function getBusinessIdField(): ?string
+    {
+        return $this->businessIdField;
+    }
+
+    public function setBusinessIdField(string $businessIdField): ContentType
+    {
+        $this->businessIdField = $businessIdField;
+        return $this;
```

Comment:
This should be discussed but in general a setter should return void.

Reply:
Until now all setter in the core are fluent